src/views/tutorial-view/components/featured-in-collections/index.tsx

```diff
@@ -1,46 +1,43 @@
-import { getCollectionSlug } from 'views/collection-view/helpers'
-import CardLink from 'components/card-link'
-import { Collection as ClientCollection } from 'lib/learn-client/types'
-
-interface FeaturedInCollectionsProps {
-  collections: CollectionCardProps[]
-}
-
-// @TODO move this to the collection card component file when we make it
```

Comment:
Moved `CollectionCard` out to a separate file, even though we're not _quite_ ready to make it yet (design file still says WIP).